Makefile

```diff
@@ -124,21 +124,13 @@ PINNED_DEPENDENCIES := \
 # Code coverage & test report output files.
 TEST_OUTPUT_ROOT        := ./.testoutput
 NEW_COVER_PROFILE       = $(TEST_OUTPUT_ROOT)/$(shell xxd -p -l 16 /dev/urandom).cover.out   # generates a new filename each time it's substituted
-SUMMARY_COVER_PROFILE  := $(TEST_OUTPUT_ROOT)/summary.cover.out
 NEW_REPORT              = $(TEST_OUTPUT_ROOT)/$(shell xxd -p -l 16 /dev/urandom).junit.xml   # generates a new filename each time it's substituted
+COVERPKG_FLAG 		    = -coverpkg=$(shell go list ./... | paste -sd "," -)
```

Comment:
Just listing all packages here. Anything we don't want to be reported is filtered out by Codecov.

Reply:
Is the alternative to write a Go script? I feel like there's already way too many Unix-specific stuff in this Makefile already. (e.g. /dev/urandom just above it)